### ISSUE TITLE
Error: java.lang.reflect.InvocationTargetException in .class files

### ISSUE THREAD

**Issue by shannah** [orfjackal/retrolambda#55]
After converting some .class files with retrolambda, the resulting .class files seem to have some problems.  If I run `javap -verbose OutputClass.class` I see the following type of output at the end of some (but not all) method definitions.

~~~~
Error: java.lang.reflect.InvocationTargetException
        StackMap: length = 0x3
         00 01 0F 
~~~~

I can't seem to identify anything special about the methods where this is happening.  Any idea what would cause this?

**Comment by orfjackal** [comment]
Please give a complete example for reproducing this.

**Comment by shannah** [comment]
Here is a more complete test to demonstrate the issue.

**Comment by orfjackal** [comment]
Thanks. This seems to be something like Retrolambda generating stack map frames even though the target bytecode is Java 5, which doesn't have them.

**Comment by shannah** [comment]
Thanks for looking at this.  Is there an easy way to strip these out?  They are making some other tools choke on the .class files later on down the line.

**Comment by orfjackal** [comment]
I made a fix for it. I'll release it later today.

**Comment by orfjackal** [comment]
This has been fixed in Retrolambda 2.0.3
